modules/python/Makefile.env

```diff
@@ -45,14 +40,14 @@ pipenv/check: python/check ; $(call assert-set,PY_PIPENV) && echo "Found pipenv:
 .PHONY: pipenv/debug
 pipenv/debug: python/debug
 	$(call print-var,PY_PIPENV)
-	$(call print-var,PY_PIPFILE_EXISTS)
-	$(call print-var,PY_CURRENT_VENV)
-	$(call print-var,PY_VENV)
+	$(call print-var,WORKON_HOME)
 	$(call print-var,PIPENV)
+	$(call print-var,PIPENV_HOME)
+	$(call print-var,PY_VENV)
 	$(call print-var,WITH_PIPENV)
 
 ## Install and activate a virtual environment using pipenv
-pipenv: pipenv/check $(PY_VENV) Pipfile.lock
+pipenv: pipenv/check $(PY_VENV) $(PIPENV_HOME)/Pipfile.lock
```

Comment:
Since `$(PY_VENV)` requires `$(PY_VENV)/bin/activate` requires `$(PIPENV_HOME)/Pipfile.lock` this target can simplify to:
```
pipenv: pipenv/check $(PY_VENV)`
```